src/coreComponents/virtualElement/ConformingVirtualElementOrder1.cpp

```diff
@@ -0,0 +1,433 @@
+#include "ConformingVirtualElementOrder1.hpp"
+
+namespace geosx
+{
+namespace virtualElement
+{
+void ConformingVirtualElementOrder1::ComputeProjectors( MeshLevel const & mesh,
+                                                        localIndex const & regionIndex,
+                                                        localIndex const & subRegionIndex,
+                                                        localIndex const & cellIndex )
```

Comment:
In the next PR when you want to call this function inside a kernel to compute the basis functions and the stabilization matrix in each cell, you may have to update the signature of this function a little bit. 

Instead of passing the `mesh` and the triplet of indices that identifies a cell, I think you would directly pass the volume and center of the particular cell, plus some `arrayViews` (say, `faceCenters`, `faceNormals`, `edgeToNodes`, etc), `arraySlices` (say, `elemToFaces` and `cellToNodes` for the particular cell) that you need inside the function. You would also pass a reference to the three arrays (basis functions, derivatives, and stabilization matrix) that you want to populate in this function. With these changes (and maybe some small others) it will be possible to call `ComputeProjectors` inside a kernel.